tests/spec/localhost/nvidia_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+has_nvidiactl = file('/dev/nvidiactl').exists?
+
+# should always be installed
+describe file('/usr/bin/nvidia-docker') do
+  it { should exist }
+end
+
+# we just can test this on gpu enabled instances.
+describe command('nvidia-docker run --rm nvidia/cuda nvidia-smi'), :if => has_nvidiactl do
+  its(:stdout) { should contain /Tesla K80/ }
```

Comment:
this would lead to errors 'cause the nvidia-docker expected devices are not in place. Thats the reason why i exclude it. 

However but i check if the command is installed. 

It might make sense to add an optional step in the `test.sh` for checking GPU instances. But there are no p2 instances in eu-central-1 so doing it all the time or exclusively adds additional constraints to the testing